orttraining/orttraining/python/training/ortmodule/__init__.py

```diff
@@ -57,7 +57,7 @@
 if not is_torch_cpp_extensions_installed(ORTMODULE_TORCH_CPP_DIR) and '-m' not in sys.argv:
     _FALLBACK_INIT_EXCEPTION = wrap_exception(
         ORTModuleInitException,
-        EnvironmentError(
+        RuntimeError(
```

Comment:
@thiagocrepaldi it seems like `OSError` is a subclass of `EnvironmentError`. From the [documentation](https://docs.python.org/3/library/exceptions.html#OSError) it seems like:
>Changed in version 3.3: EnvironmentError, IOError, WindowsError, socket.error, select.error and mmap.error have been merged into OSError, and the constructor may return a subclass.

Even if the `EnvironmentError` constructor returns an exception of `OSError`, it should be caught by `except EnvironmentError`. I am not sure if this indeed is the reason for the try catch not catching the exception.